tests/MongoDB.EntityFrameworkCore.FunctionalTests/LoggingTests.cs

```diff
@@ -104,4 +104,67 @@ public void Query_writes_event_via_LoggerFactory_without_mql_when_no_sensitive_l
         Assert.Contains($"{DbName}.moons.aggregate([?])", message);
         Assert.DoesNotContain("yearOfDiscovery", message);
     }
+
+    [Fact]
+    public void BulkWrite_writes_log_via_LogTo_with_counts()
+    {
+        List<string> logs = [];
+
+        using var guidesFixture = new SampleGuidesFixture();
+        var db = GuidesDbContext.Create(guidesFixture.MongoDatabase, s =>
+        {
+            logs.Add(s);
+            testOutputHelper.WriteLine(s);
+        });
+
+        db.Planets.RemoveRange(db.Planets.Where(m => m.name.StartsWith("M")));
+        foreach (var planet in  db.Planets.Where(m => m.hasRings))
+        {
+            planet.hasRings = false;
+        }
+        db.Planets.Add(new Planet
+        {
+            name = "Proxima Centauri d", hasRings = false, orderFromSun = -1
+        });
+
+        Assert.Equal(7, db.SaveChanges());
+
+        var log = Assert.Single(logs, l => l.Contains("MongoEventId.ExecutedBulkWrite"));
+
+        Assert.Contains("Executed Bulk Write", log);
+        Assert.Contains($"Collection='{guidesFixture.MongoDatabase.DatabaseNamespace.DatabaseName}.planets'", log);
```

Comment:
I'm confused how this assertion succeeds given the spelling error in the format string.

Reply:
The format specifiers are actually positional rather than by name so they're for information only.